coursedata/adventures/de.yaml

```diff
@@ -1899,7 +2047,7 @@ adventures:
                     print I need to get out of here!
                     print There are 3 doors in front of me..
                     ask Which door should i pick?
-                    echo I choose door 
+                    echo I choose door
```

Comment:
```suggestion
                    echo I choose door 
```

Reply:
Same here. I suggest filing an issue to correct it in the EN language file. I'd suppose that it then finds its way into other language files as well.